site/layouts/partials/fancymarkdown.html

```diff
@@ -1 +1,11 @@
-{{ . | replaceRE "(<h[1-9] id=\"([^\"]+)\".+)(</h[1-9]+>)" `${1}<a href="#${2}" class="hanchor" ariaLabel="Anchor"> 🔗&#xFE0E;</a> ${3}` | safeHTML }}
\ No newline at end of file
+{{ if not (eq .Site.Title .Params.title) }}
+<h1>{{ .Params.title }}</h1>
+{{ end }}
+
```

Comment:
we're injecting blank lines if these aren't set?

Reply:
I guess so, but have hopefully covered all main pages. Not sure how to make it mandatory with Hugo.

Reply:
we can put the blank lines inside the blocks before the `{{ end }}`